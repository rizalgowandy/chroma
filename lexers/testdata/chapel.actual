config const numMessages = 100;

forall msg in 1..numMessages do
  writeln("Hello, world! (from iteration ", msg, " of ", numMessages, ")");

// Oh, and a comment for good measure.
