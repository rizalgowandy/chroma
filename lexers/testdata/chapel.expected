[
  {"type":"KeywordDeclaration","value":"config"},
  {"type":"TextWhitespace","value":" "},
  {"type":"KeywordDeclaration","value":"const"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameOther","value":"numMessages"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Operator","value":"="},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralNumberInteger","value":"100"},
  {"type":"Punctuation","value":";"},
  {"type":"TextWhitespace","value":"\n\n"},
  {"type":"Keyword","value":"forall"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameOther","value":"msg"},
  {"type":"TextWhitespace","value":" "},
  {"type":"KeywordDeclaration","value":"in"},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralNumberInteger","value":"1"},
  {"type":"Operator","value":".."},
  {"type":"NameOther","value":"numMessages"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Keyword","value":"do"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"NameOther","value":"writeln"},
  {"type":"Punctuation","value":"("},
  {"type":"LiteralString","value":"\"Hello, world! (from iteration \""},
  {"type":"Punctuation","value":","},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameOther","value":"msg"},
  {"type":"Punctuation","value":","},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralString","value":"\" of \""},
  {"type":"Punctuation","value":","},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameOther","value":"numMessages"},
  {"type":"Punctuation","value":","},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralString","value":"\")\""},
  {"type":"Punctuation","value":");"},
  {"type":"TextWhitespace","value":"\n\n"},
  {"type":"CommentSingle","value":"// Oh, and a comment for good measure.\n"}
]
